### ISSUE TITLE
Restyle links on home page tiles

### ISSUE THREAD

**Issue by joeliedtke** [kubeflow/website#1211]
Currently the links on the home page tiles have low contrast from the page text, which can be a usability and accessibility issue. Please update the link style to correct this issue. For example, please add the bold style to _index.html line 173:

```
 a.text-light{
	text-decoration: underline dotted paleturquoise;
        font-weight: bold;
  }
```

**Comment by simmscg** [comment]
I think this might be a bit more complicated than at first glance.

- Setting everything to bold removes the capability to selectively add bold to text within a paragraph.
- The font weights for Open Sans aren't being loaded correctly, so we can't select an intermediate weight.
- Contrast fails WCAG AA at normal text sizes, but the complication isn't the text color, it's the background color, which needs to be darker than the primary brand color.

I'm working on something that addresses all of this, but be prepared for colors to shift a bit.
